Alarms.h

```diff
@@ -222,11 +227,17 @@ class AlarmManager {
     NO_TIDAL_PR,
     OVER_CURREN,
     MECH_FAILUR,
-    NOT_CONFIRM,
+    NOT_CONFIRM_TV,
+    NOT_CONFIRM_RR,
+    NOT_CONFIRM_IE,
+    NOT_CONFIRM_AC,
```

Comment:
Why do we need separate alarms for each not_confirm? The old way was better since it's really the same alarm